### ISSUE TITLE
Unexpected error during attach new image if quotas limit is reached

### ISSUE THREAD

**Issue by kvaps** [OpenNebula/one#3610]
**Description**
When user reached his system disks size quota and tries to attach new image, he will see an error:

```
Unexpected error executing code for particular method, detected by Xmlrpc-c method registry code. Method did not fail; rather, it did not complete at all. Assigning to already instantiated xmlrpc_c::value
```

**To Reproduce**
* Create new Group
* Create new VM
* Chgrp for this VM
* Add group Quotas for System disks
* Try to overuse it

**Expected behavior**

Reliable error message, that quota limit for system disks has reached
 
**Details**
 - Affected Component: Oned
 - Hypervisor: KVM
 - Version: 5.8.0-5.8.4

**Additional context**

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by kvaps** [comment]
I just wanted to append that problem still persist on 5.10.2

![image](https://user-images.githubusercontent.com/7556217/75439142-94b3e380-5959-11ea-9bbb-3672008d9449.png)

**Comment by kvaps** [comment]
xml method call, just for the confirmation that it is oned problem, not sunstone

```xml
<!---------- INPUT ------------>

<?xml version="1.0"?>
<methodCall>
    <methodName>one.vm.attach</methodName>
    <params>
        <param><value>oneadmin:deadbeefdeadbeefdeadbeefdeadbeef</value></param>
        <param><value><i4>419</i4></value></param>
        <param><value><string>DISK=[SIZE=1024, IMAGE_ID=210]</string></value></param>
    </params>
</methodCall>

<!--------- OUTPUT ------------>

<?xml version="1.0" encoding="UTF-8"?>
<methodResponse>
<fault>
<value><struct>
<member><name>faultCode</name>
<value><i4>-500</i4></value></member>
<member><name>faultString</name>
<value><string>Unexpected error executing code for particular method, detected by Xmlrpc-c method registry code.  Method did not fail; rather, it did not complete at all.  Assigning to already instantiated xmlrpc_c::value</string></value></member>
</struct></value>
</fault>
</methodResponse>

<!---------- END -------------->
```

**Comment by sergiojvg** [comment]
Same issue here, thanks to this issue I checked the quota, thanks @kvaps. Running 5.10.2

**Comment by paczerny** [comment]
Fixed in 6.0.3
